.changeset/drawer-focus.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': patch
+---
+
+fix(drawer) Added tabIndex for focusing content when opened.
```

Comment:
We should update this now that the implementation has changed

Reply:
"I see. Does this work as we expect if someone passes isFocused to the second or third tab child?" This works on any NavTab as well indeed.